scripts/after_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+export PATH=$PATH:/home/ubuntu/.cargo/bin
+
+APP_NAME=$(echo $APPLICATION_NAME | awk -F- '{ print $2 }')
+BRANCH=$(curl http://169.254.169.254/latest/meta-data/tags/instance/$APP_NAME)
+ 
+# Prepare directory
+sudo rm -rf ~/$APP_NAME/tlsn
+sudo mv ~/tlsn/ ~/$APP_NAME
+sudo mkdir -p ~/$APP_NAME/tlsn/notary-server/target/release
+sudo chown -R ubuntu.ubuntu ~/$APP_NAME
+ 
+git clone -b $BRANCH --no-checkout https://github.com/tlsnotary/tlsn.git /tmp/tlsn_remove
```

Comment:
Can we add a comment here very briefly explaining why this step is needed?

P/S: there is a slim chance that this copied .git folder of the dev branch has included a newer commit that is not part of the built binary

Reply:
If you are ok with this: https://github.com/tlsnotary/tlsn/pull/419#discussion_r1469599960
i can also migrate this part to ci.yml workflow.

Reply:
Either what you suggested or if you agree with this https://github.com/tlsnotary/tlsn/pull/419#discussion_r1469599960
i can migrate the .github part to ci.yml workflow.

Either what you suggested or if you agree with this, i can migrate the .github part to ci.yml workflow.

Reply:
Either what you suggested or if you agree with this https://github.com/tlsnotary/tlsn/pull/419#discussion_r1469599960
i can migrate the .github part to ci.yml workflow.

Reply:
let's go with my original suggestion if possible, so that the ci.yml workflow is kept simple and not too complicated :)

Reply:
Done! (https://github.com/ntampakas/tlsn/commit/022b491c1e8f92429dc430512167634f625489b2)